core-api/src/main/java/com/optimizely/ab/internal/ExperimentUtils.java

```diff
@@ -66,14 +66,11 @@ public static boolean isUserInExperiment(@Nonnull ProjectConfig projectConfig,
             return true;
         }
 
-        // if there are audiences, but no user attributes, the user is not in the experiment.
-        if (attributes.isEmpty()) {
-            return false;
-        }
-
         for (String audienceId : experimentAudienceIds) {
             Condition conditions = projectConfig.getAudienceConditionsFromId(audienceId);
-            if (conditions.evaluate(attributes)) {
+            Boolean conditionEval = conditions.evaluate(attributes);
+
+            if (conditionEval != null && conditionEval){
```

Comment:
Can this be shortened to if `conditionEval == true`?

Reply:
Need to check for null.  Auto boxing I believe would throw an exception here.

Reply:
OK. 

Also, nit: space before the opening `{`